src/schemas/v2/combined.json

```diff
@@ -36814,184 +36814,236 @@
-                ]
+              "estimatedPrice": {
+                "title": "Estimated Price",
+                "description": "Approximation of the property's current market value",
+                "type": "number"
               },
-              "BasicDetails": {
-                "title": "Basic Property Details",
-                "type": "object",
-                "properties": {
-                  "bedrooms": {
-                    "title": "Number of bedrooms",
-                    "type": "integer"
-                  },
-                  "bathrooms": {
-                    "title": "Number of bathrooms",
-                    "type": "integer"
-                  },
-                  "yearBuilt": {
-                    "title": "Built Year",
-                    "type": "string",
-                    "format": "date"
-                  },
-                  "floorArea": {
-                    "title": "Floor area",
-                    "type": "object",
-                    "properties": {
-                      "inSquareMeter": {
-                        "type": "object",
-                        "properties": {
-                          "area": {
-                            "title": "Area in sqaure meters",
-                            "type": "number"
-                          },
-                          "pricePerSquareMeter": {
-                            "title": "Price per square meter",
-                            "type": "number"
-                          }
+              "yield": {
+                "title": "Yield",
+                "description": "Return on investment (ROI) generated by an income-producing asset",
+                "type": "number"
+              }
+            }
+          }
+        },
+        "if": {
+          "properties": {
+            "listingType": {
+              "const": "Recently Sold"
+            }
+          }
+        },
+        "then": {
+          "properties": {
+            "listingStatus": {
+              "title": "Listing Status ",
+              "description": "It provides specific information about the status of a property listing",
+              "type": "object",
+              "properties": {
+                "dateSold": {
+                  "title": "Sold Date",
+                  "type": "string",
+                  "format": "date-time"
+                },
+                "priceSold": {
+                  "title": "Sold Price",
+                  "type": "object",
+                  "properties": {
+                    "amount": {
+                      "title": "Price",
+                      "type": "number"
+                    },
+                    "currency": {
+                      "title": "Currency",
+                      "type": "object",
+                      "properties": {
+                        "type": {
+                          "type": "string",
+                          "enum": ["USD", "EUR", "GBP", "JPY", "CAD", "AUD", "Other"]
```

Comment:
So I guess this is the UK Property Data Trust Framework schema and we only represent the price of the actual property in GBP, would we expect comparable properties to be priced other than in GBP?

Reply:
yes majorly only GBP property pricing is under consideration, so removing currency and kept only salePrice and soldPrice as plain properties